### PR TITLE
ethernet support for NUCLEO_F746ZG

### PR DESCRIPTION
**Pull request by cfpena**
Adding ethernet support using PR #4541
Append SSL to settings

### CODE REVIEW

File: ports/stm32/boards/NUCLEO_F746ZG/mpconfigboard.mk [hunk 1]
```diff
@@ -4,3 +4,6 @@ AF_FILE = boards/stm32f746_af.csv
 LD_FILES = boards/stm32f746.ld boards/common_ifs.ld
 TEXT0_ADDR = 0x08000000
 TEXT1_ADDR = 0x08020000
+
+# MicroPython settings
+MICROPY_PY_LWIP = 1
```
Comment (dpgeorge) [inline ports/stm32/boards/NUCLEO_F746ZG/mpconfigboard.mk]: Following other boards with ETH, I think it'd be good to also enable the `ussl` module via:
```
MICROPY_PY_USSL = 1
MICROPY_SSL_MBEDTLS = 1
```

Comment (cfpena) [reply ports/stm32/boards/NUCLEO_F746ZG/mpconfigboard.mk]: It sounds good, I will add it

Comment (cfpena) [reply ports/stm32/boards/NUCLEO_F746ZG/mpconfigboard.mk]: > Following other boards with ETH, I think it'd be good to also enable the `ussl` module via:
> 
> ```
> MICROPY_PY_USSL = 1
> MICROPY_SSL_MBEDTLS = 1
> ```

Done

**Comment by dpgeorge** [comment]
Thanks for updating, merged in cfec0540732b6ce3e21d3a81f088dc6a328ff4d7

**Pull request closed without merging**